app/src/main/java/edu/grinnell/appdev/grinnelldirectory/DBAPICaller.java

```diff
@@ -21,13 +17,10 @@
 
 public class DBAPICaller implements SearchCaller {
 
-    private static final String BASE_URL = "https://appdev.grinnell.edu/api/db/v1/";
-    private static final int RESPONSE_FORBIDDEN = 403;
+    private static final String BASE_URL = "https://www.cs.grinnell.edu/~pandeyan/appdev/";
```

Comment:
For time being, are you going to use this url instead of the primary Appdev one?

Reply:
Yes, we are using this for testing until the appdev server starts working.